api/admin.go

```diff
@@ -365,11 +408,24 @@ func (a *API) adminUserCreate(w http.ResponseWriter, r *http.Request) error {
 			}
 		}
 
+		if params.BanDuration != "" {
```

Comment:
```suggestion
		if params.BanDuration != "" && params.BanDuration != "none" {
```

Might be missing something here but do we take any action here if the BanDuration is none?

Reply:
yeah if BanDuration is none, then we lift the ban